build.py

```diff
@@ -9,17 +9,29 @@
 def create_parser():
     parser = argparse.ArgumentParser(description="Build PEP documents")
     # alternative builders:
-    parser.add_argument("-l", "--check-links", action="store_true")
-    parser.add_argument("-f", "--build-files", action="store_true")
-    parser.add_argument("-d", "--build-dirs", action="store_true")
+    parser.add_argument("-l", "--check-links", action="store_true",
+                        help='Checks validity of links within PEP sources. '
+                             'Cannot be used with "-f" or "-d".')
+    parser.add_argument("-f", "--build-files", action="store_true",
+                        help='Renders PEPs to "pep-XXXX.html" files (Default). '
+                             'Cannot be used with "-d" or "-l".')
+    parser.add_argument("-d", "--build-dirs", action="store_true",
+                        help='Renders PEPs to "index.html" files within "pep-XXXX" directories. '
+                             'Cannot be used with "-f" or "-l".')
 
     # flags / options
-    parser.add_argument("-w", "--fail-on-warning", action="store_true")
-    parser.add_argument("-n", "--nitpicky", action="store_true")
-    parser.add_argument("-j", "--jobs", type=int, default=1)
+    parser.add_argument("-w", "--fail-on-warning", action="store_true",
+                        help="Fails Sphinx on warnings.")
```

Comment:
```suggestion
                        help="Fail the Sphinx build on warnings.")
```

Use standard tense and describe a little more clearly